recipes/freetype/all/conanfile.py

```diff
@@ -122,6 +134,6 @@ def package_info(self):
         self.env_info.FT2_CONFIG = freetype_config
         self.user_info.LIBTOOL_VERSION = self._libtool_version
         self._chmod_plus_x(freetype_config)
-        self.cpp_info.names['cmake_find_package'] = 'Freetype'
-        self.cpp_info.names['cmake_find_package_multi'] = 'Freetype'
-        self.cpp_info.names['pkg_config'] = 'freetype2'
+        self.cpp_info.names["cmake_find_package"] = "Freetype"
+        self.cpp_info.names["cmake_find_package_multi"] = "freetype" # TODO: CMake config file has a non namespaced imported target
```

Comment:
¿????? Targets generated by `cmake_find_package` and `cmake_find_package_multi` should be the same... what's happening here? Is it a bug in Conan client?

Reply:
Official module file is `FindFreetype.cmake`, while official config file is `freetype-config.cmake`, therefore `cmake_find_package` and `cmake_find_package_multi` shouldn't be the same.

Reply:
OMG! .... so 
 * `find_package(Freetype)` using `MODULE` mode, but
 * `find_package(freetype NO_MODULE)` or `find_package(freetype CONFIG)`

---

Right now I'm ~~sad 😭~~ confused. I'm not sure if we could ever merge these two generators into one...

Reply:
When you think Protobuf is the only weird project based on cmake modules ...